client/src/components/EditMenu.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+import MenuForm from './MenuForm';
+import CatForm from './CatForm';
+
+import { Link } from 'react-router-dom';
+import Categories from './Categories';
+
+class EditMenu extends React.Component {
+  state = { menu: {}, categories: [], editName: false, addCat: false }
+
+  componentDidMount() {
+    axios.get(`/api/menus/${this.props.match.params.id}`)
+      .then( res => {
+        this.setState({ menu:res.data })
+      })
+  }
+
+  deleteMenu = (id, e) => {
```

Comment:
the delete function does not need e, only the id. Form submission functions are the ones that need e